test/types/bytes/multiplication.chpl

```diff
@@ -0,0 +1,7 @@
+writeln(b"Hello! " * 3);
+writeln(3 * b"Hello! ");
+
+writeln(b"" * 3);
+writeln(3 * b"");
+
+writeln(b"" * 0);
```

Comment:
Could you also test a non-empty string multiplied by 0? I see that you have it for the bytes type but not for this one.

Reply:
@e-kayrakli I Improved my tests, Please have a look.
`test/types/string/multiplication.chpl`
>writeln("Hello !" * 5);
writeln(5 * "Hello !");
writeln("Result:" , "" * 4);
writeln("Result:" , 4 * "");
writeln("Result:" , "" * 0);
writeln("Result:" , 0 * "");
writeln("Result:" , "Hello !" * 0);
writeln("Result:" , 0 * "Hello !");
 
`test/types/bytes/multiplication.chpl`
>writeln(b"Hello! " * 3);
writeln(3 * b"Hello! ");
writeln("Result:" , b"" * 3);
writeln("Result:" , 3 * b"");
writeln("Result:" , b"" * 0);
writeln("Result:" , 0 * b"");
writeln("Result:" , b"Hello!" * 0);
writeln("Result:" , 0 * b"Hello!");